Pipfile

```diff
@@ -6,9 +6,10 @@ verify_ssl = true
 [dev-packages]
 pytest = "*"
 tox = "*"
-flake8 = "*"
-black = "*"
+flake8 = "==3.8.4"
+black = "==20.8b1"
 pre-commit = "*"
+pytest-rerunfailures = "*"
```

Comment:
Shouldn't this be part of https://github.com/pytest-dev/pytest-html/pull/414 and not this PR?

Reply:
Indeed! Copypasta error on my part.

Reply:
Done.